jscomp/core/res_compmisc.ml

```diff
@@ -27,7 +27,7 @@ let init_path () =
   let exp_dirs =
     List.map (Misc.expand_directory Config.standard_library) dirs
   in
-  Config.load_path := List.rev_append exp_dirs [];
+  Config.load_path := List.rev_append exp_dirs [Config.standard_library];
```

Comment:
Interesting. Presumably this must have been there a long time ago.
To double check: this should have no effect on pervasives I imagine. Just in case the fact that we build 2 versions of pervasives for uncurried and legacy mode is relevant.

Reply:
Without this, the CMI files (that were formerly loaded from the cache) are not found.